### DISCUSSION TITLE
notebook/notebooks/dm/9

### DISCUSSION THREAD

**Discussion by giscus** [NoughtQ/notebook#7] [Announcements]
# notebook/notebooks/dm/9

https://noughtq.github.io/notebook/notebooks/dm/9.html

**Comment by lxazjk** [comment]
不是所有的(a, a) 都有对应的(a, b) (b, a) 所以那道题 不一定是reflexive

**Reply by NoughtQ** [reply]
不好意思，我不清楚你说的是哪一部分的内容，您能指出是哪一道题吗（最好截个图），谢谢！
